rts/rts.c

```diff
@@ -39,6 +40,54 @@ int opterr = 0;
 char rts_verbose = 0;
 char rts_debug = 0;
 
+long num_wthreads;
+
+char *rts_mon_path = NULL;
+
+struct wt_stat {
+    unsigned int idx;          // worker thread index
+    char key[10];              // thread index as string for convenience
```

Comment:
We should expose the env.argv[0], i.e. the name of the acton program that we are currently running. Maybe PID too? This way it'll be easier to follow in graphs. Maybe that shouldn't be under the WTS command though? Damn convenient though...

Reply:
I've added argv[0] and the pid to the exported data. We're not really doing anything with it in actonmon for now... I want to play around a little more with it but first I want this PR merged, so leaving it like this for now. The future changes are likely in actonmon only and not to the on-the-wire format.